CONTRIBUTING.md

```diff
@@ -51,29 +51,30 @@ npm run unit-test
 
 ```
 export TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
+export TEST_SESSION_TOKEN=<YOUR_SESSION_TOKEN>
 npm run integration-test
 ```
 
-### Run all tests
+How do I get a session token?
+
+Visit the [preprod console](https://momento-console-preview.vercel.app/caches), once logged in, 
+inspect the webpage, navigate to the `Storage` tab and and under `Local Storage`,
+copy your `momento_session_token` :tada:
 
-```
-export TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
-npm run test
-```
 
-### Run auth integration tests
```

Comment:
I think we still need this, since you have reverted changes to add auth tests to the regular npm test task.

Reply:
Oh! Good catch